src/app/public/modules/list-view-checklist/list-view-checklist.component.ts

```diff
@@ -353,9 +353,15 @@ export class SkyListViewChecklistComponent extends ListViewComponent implements
         this.dispatcher.filtersUpdate(filters);
       });
 
-    this.dispatcher.next(
-      new ListPagingSetPageNumberAction(Number(1))
-    );
+      if (isSelected) {
```

Comment:
Would you care to put some in-code comments here describing why this is being done. I think this is a spot where we won't remember why this specific code is here.

Reply:
good suggestion. done.